src/ethereum_spec_tools/forks.py

```diff
@@ -61,7 +85,15 @@ def discover(cls: Type[H]) -> List[H]:
         forks: List[H] = []
 
         for pkg in modules:
-            mod = importlib.import_module(pkg.name)
+            if isinstance(pkg.module_finder, importlib.abc.MetaPathFinder):
+                found = pkg.module_finder.find_module(pkg.name, None)
+            else:
+                found = pkg.module_finder.find_module(pkg.name)
+
+            if not found:
+                raise Exception(f"unable to load module {pkg.name}")
+
+            mod = found.load_module(pkg.name)
```

Comment:
@SamWilsn After some investigation, I have narrowed it down and it seems to me that these changes (L88 - L96) are causing a memory bloat while loading and running evm tools tests. Not entirely sure why though. Would you have any thoughts?